script/deploy/healthCheck.ts

```diff
@@ -256,44 +256,61 @@ const main = defineCommand({
     }
 
     //          ╭─────────────────────────────────────────────────────────╮
-    //          │                   Check approved DEXs                   │
+    //          │                   Check whitelisted addresses           │
     //          ╰─────────────────────────────────────────────────────────╯
-    if (dexs) {
-      consola.box('Checking DEXs approved in diamond...')
+    if (expectedWhitelistedAddresses) {
+      consola.box('Checking whitelisted addresses in diamond...')
 
-      // connect with diamond to get whitelisted DEXs
-      const dexManager = getContract({
+      // connect with diamond to get whitelisted addresses
+      const whitelistManager = getContract({
         address: deployedContracts['LiFiDiamond'],
         abi: parseAbi([
-          'function approvedDexs() external view returns (address[])',
-          'function isFunctionApproved(bytes4) external returns (bool)',
+          'function approvedDexs() external view returns (address[])', // old DexManagerFacet (depricated replaced by WhitelistManagerFacet getWhitelistedAddresses)
```

Comment:
Should probably just use the new methods and ensure the old methods no longer work.

Reply:
updated